CMakePresets.json

```diff
@@ -11,6 +11,26 @@
       "hidden": true,
       "binaryDir": "${sourceDir}/build/${presetName}",
       "generator": "Ninja"
+    },
+	{
+      "name": "xcode-base",
+      "hidden": true,
+      "binaryDir": "${sourceDir}/build/${presetName}",
+      "generator": "Xcode"
+    },
+	{
+      "name": "gcc-base",
+      "hidden": true,
+      "cacheVariables": {
+        "CMAKE_CXX_COMPILER": {
+          "type": "STRING",
+          "value": "/usr/bin/gcc"
+        },
+        "CMAKE_C_COMPILER": {
+          "type": "STRING",
+          "value": "/usr/bin/gcc"
+        }
```

Comment:
Ah OK, good spot. 
It works in my case - if I don't specify this, then it definitely uses AppleClang by default, because VCPKG then pipes up and says the version of AppleClang in OSX <10.15 isn't supported. With this, VCPKG is happy.
Not sure how to make sure this definitely points to GCC. Do you have any ideas?